dist/js/wavesurfer-angular.js

```diff
@@ -62,8 +65,11 @@
                             }
                         });
                     };
+
+                    var waveform = element.children()[0].children[0].children[4];
```

Comment:
This is pretty ugly, but to be able to find that element and pass it to wavesurfer I don't really know if there is a better way. The unique id is placed into the id after this function gets called. Also angular does not support doing element.children("SELECTOR > SELECTOR") so you have to navigate in this way.